MapleServer2/Types/Mob.cs

```diff
@@ -25,5 +17,12 @@ public Mob(int id)
             Stats = mob.Stats;
             Friendly = mob.Friendly;
         }
+
+        public void UpdateStats(double damage)
+        {
+            Mob mob = this;
```

Comment:
You don't need this line, you should be able to just access `Stats` and `IsDead` directly here

Reply:
sure! yeah, i though i need the same value where is used but is in the same class doesn't need it.